app/ml_models/rnn/loaded_rnn_model.py

```diff
@@ -1,6 +1,25 @@
 import os
 import torch
-from app.ml_models.rnn.rnn_model import RNN
+from app.ml_models.rnn.rnn_model import RNN, RNNAnna
+from app.ml_models.rnn.data_tools import WordLevelDataset
+
+
+def load_model(model_name: str = '2021_straattaal_epoch100.pt', device: str = 'cpu'):
+    """
+    Args
+        model_name: Filename of the model
+        device: CUDA device name to map to, probably cpu
+    """
+    path = os.path.join(os.path.abspath(os.getcwd()),
+                        "app", "ml_models", "rnn")
+    path = os.path.join(path, model_name)
+    dataset = WordLevelDataset('data/', 'straattaal.txt')
+    # TODO: Fix hardcoded hidden size
+    m = RNNAnna(dataset.vocabulary_size, 128)
```

Comment:
Maybe as parameter to function name, e.g. based on the filename of the model in which we mention the hidden size?

Reply:
Or just indeed derive it from a(n appropriate) filename.

Reply:
It would be better to store in in the file with the state_dict maybe. Or even infer it from the RNN data from the file, but that is uglier.

Do you want me to do this for this PR?

Reply:
Let's add it as an issue instead and update in another, smaller PR

Reply:
#31